platform/ios/src/MGLAnnotationView.mm

```diff
@@ -239,7 +239,10 @@ - (void)setDragState:(MGLAnnotationViewDragState)dragState animated:(BOOL)animat
     }
     else if (dragState == MGLAnnotationViewDragStateCanceling)
     {
-        NSAssert(self.annotation, @"Annotation property should not be nil.");
+        if (!self.annotation) {
+            [NSException raise:@"MGLInvalidAnnotationException"
```

Comment:
Let’s stick to standard exception names, except for similar exceptions strewn throughout the codebase, such as `MGLAbstractClassException`. In this case, `NSInvalidArgumentException` seems reasonable. (We are using ad-hoc names in other places, but we should clean them up at some point.)